Modules/Microsoft365DSC/DSCResources/MSFT_EXOServicePrincipal/settings.json

```diff
@@ -0,0 +1,24 @@
+{
+    "resourceName": "EXOServicePrincipal",
+    "description": "Use the New-ServicePrincipal cmdlet to create service principals in your cloud-based organization.",
```

Comment:
Can we make the description related to the resource and what it does instead of having it generic for the New cmdlet?